src/jobs/megalinter_test_deploy.yml

```
@@ -0,0 +1,180 @@
+      The resource class to use for the job.
+      Available options:
+      - small: 1 vCPU, 2GB RAM
+      - medium: 2 vCPUs, 4GB RAM
+      - medium+: 3 vCPUs, 6GB RAM
+      - large: 4 vCPUs, 8GB RAM
+      - xlarge: 8 vCPUs, 16GB RAM
+      - 2xlarge: 16 vCPUs, 32GB RAM
+      - 2xlarge+: 20 vCPUs, 40GB RAM
+      See https://circleci.com/docs/2.0/configuration-reference/#resource_class
+    persist_megalinter_reports:
+    type: boolean
```

yamllint (1.35.1)

[error] duplication of key "type" in mapping

(key-duplicates)